dali/pipeline/operators/op_spec.h

```diff
@@ -474,6 +503,9 @@ inline bool OpSpec::TryGetArgumentImpl(
     if (ws == nullptr)
       return false;
     const auto& value = ws->ArgumentInput(name);
+    if (value.shape() != kernels::uniform_list_shape(GetArgument<int>("batch_size"), {1})) {
```

Comment:
Sorry, we can't have it - this is FAR too expensive for a simple check, executed on a per-sample basis.

Reply:
I reused the logic from GetArgument.